recipes/django-nose/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "django-nose" %}
+{% set version = "1.4.4" %}
+{% set sha256 = "c0b904927fcc2f9d8c55ad1afa18c2e77d74f44ef162c35e07930af5a73ba4ba" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - nose >=1.2.1
+
+  run:
+    - python
+    - django >=1.8,<1.11
+    - nose >=1.2.1
+
+test:
+  imports:
+    - django_nose
+    - django_nose.management
+    - django_nose.management.commands
+
+about:
+  home: http://github.com/django-nose/django-nose
+  license: BSD License
```

Comment:
Please drop `License` and specify number of clauses. Believe this is `BSD 3-Clause`.